loadtest/user/userentity/actions.go

```diff
@@ -318,7 +318,7 @@ func (ue *UserEntity) GetChannelsForTeam(teamId string) error {
 	if err != nil {
 		return err
 	}
-	channels, resp := ue.client.GetChannelsForTeamForUser(teamId, user.Id, "")
+	channels, resp := ue.client.GetChannelsForTeamForUser(teamId, user.Id, false, "")
```

Comment:
Same as below, let's have the controller decide whether to include or not deleted channels.

Reply:
Agree